examples/graph_data_demo.ipynb

```diff
@@ -139,17 +174,168 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "source": [
+    "## Save and Load\n",
+    "Below you will see an example of how a Graph Profile can be saved and loaded again."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# The default save filepath is profile-<datetime>.pkl\n",
+    "profile.save(filepath=\"profile.pkl\")\n",
+    "\n",
+    "new_profile = dp.GraphProfiler.load(\"profile.pkl\")\n",
+    "new_report = new_profile.report()"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "pp.pprint(report)"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Diff\n",
```

Comment:
let's follow the pattern of other notebooks and make these more like phrases:
`Differences in Data`